components/rmrk/ExploreLayout/ExploreLayout.vue

```diff
@@ -1,11 +1,17 @@
 <template>
   <div>
-    <b-tabs v-model="selectedTab" size="is-small">
-      <b-tab-item label="GALLERY" value="GALLERY"><Gallery /></b-tab-item>
-      <b-tab-item label="COLLECTION" value="COLLECTION">
+    <div v-if="$route.query.search" class="mb-3 is-flex is-align-items-center">
+      Showing results for
+      <span class="text__stroked search-term ml-3">{{
+        $route.query.search
+      }}</span>
+    </div>
```

Comment:
I would change this by
```suggestion
    <div v-if="$route.query.search" class="block">
      Showing results for
      <span class="text__stroked is-size-3">{{
        $route.query.search
      }}</span>
    </div>
```